schema/materials_category/common/supercell_matrix.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
should be supercell_matrix_2d, supercell_matrix_3d and can be in primitive - it probably already is there. 

scaling_factor -> repetitions

we can have a folder in `primitive` call it `crystal` and put parameters there

thickness -> `crystal/slab/thickness`
vacuum -> `crystal/slab/vacuum`